integration-tests/CheckMate.postman_collection.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
This postman collection can be copypasta-ed to postman, edited and updated in the repo so that it will run during the CI test